### PR TITLE
Adding more logs when reading/writing meta file

### PR DESCRIPTION
**Pull request by lipingxue**
This PR added more logs when reading/writing meta file to triage #1371 . Please review it.

### CODE REVIEW

File: esx_service/utils/kvESX.py [hunk 1]
```diff
@@ -256,32 +257,42 @@ def load(volpath):
     """
     Load and return dictionary from the sidecar
     """
+    logging.info("Load dictionary from sidecar for vol path %s", volpath)
```
Comment (msterin) [inline esx_service/utils/kvESX.py]: it feels like a debug, not info. Can you elaborate why do we want it on every log all the time ?

File: esx_service/utils/kvESX.py [hunk 2]
```diff
@@ -256,32 +257,42 @@ def load(volpath):
...
+                logging.exception("load:Failed to access %s", meta_file)
                 return None
 
     try:
         return json.loads(kv_str)
     except ValueError:
         # Adding this log for DEBUG
-        logging.warning("kv_str from meta file is %s ", kv_str)
-        logging.exception("Failed to decode meta-data for %s", volpath)
+        logging.warning("load:Dump kv_str from meta file")
```
Comment (msterin) [inline esx_service/utils/kvESX.py]: Do we really need 3 lines in log for that ? Can it be compressed into 1 ?

File: esx_service/utils/kvESX.py (hunk 1 above)
Comment (lipingxue) [inline esx_service/utils/kvESX.py]: Mark, the reason I put it as info is the failure cannot hit every time, that is why I want to merge it to master so that every CI run will able to log the enough information that I need. I have more chance to reproduce this issue with needed log.

Comment (msterin) [inline esx_service/utils/kvESX.py]: so this is for a bug hunting ? Then there should be an issue to roll it back when the bug is found. 

I do not think we should print that much information which really has no value outside of catching this bug.

File: esx_service/utils/kvESX.py [hunk 3]
```diff
@@ -298,7 +310,15 @@ def save(volpath, kv_dict):
     vol_name = vmdk_utils.get_volname_from_vmdk_path(volpath)
     while True:
         try:
+            if os.stat(meta_file).st_size > 0:
```
Comment (pshahzeb) [inline esx_service/utils/kvESX.py]: I think this block of logging is redundant.



Agreed that this logs are temp and would be removed once we hit the root cause of the issue about the volume metadata  being unavailable(either corrupted or empty); do you mind wrapping this lines into a function and call it twice. The only reason being this change is being merged into master.

File: esx_service/vmdk_ops.py [hunk 4]
```diff
@@ -1474,7 +1474,7 @@ def set_vol_opts(name, tenant_name, options):
 
     vmdk_path = vmdk_utils.get_vmdk_path(path, vol_name)
 
-    logging.debug("set_vol_opts: path=%s vmdk_path=%s", path, vmdk_path)
+    logging.info("set_vol_opts: path=%s vmdk_path=%s", path, vmdk_path)
```
Comment (pshahzeb) [inline esx_service/vmdk_ops.py]: Essentially  this log (and the related ones) are meant to be debug itself.

Do you mind registering the issue to revert this change ?(either due to test issue or product issue)

**Comment by pshahzeb** [comment]
Contingent to CI pass and no merge conflicts (not expecting any though)

**Comment by shuklanirdesh82** [comment]
CI failure: https://ci.vmware.run/vmware/docker-volume-vsphere/548

@lipingxue  tests from test-esx are failing.

File: esx_service/utils/kvESX.py (hunk 1 above)
Comment (lipingxue) [inline esx_service/utils/kvESX.py]: #1419 has been filed to track removing the log added in this PR after the root cause of #1371 is found.
